docs/flow-setup-tutorial.md

```diff
@@ -0,0 +1,34 @@
+# Flow setup Tutotrial
+
+Flow is a static-type checker for Javascript code
+
+## The steps to install Flow correctly from scratch are
+
+####  Installation tip : install flow in the root folder of zulip-mobile
+
+####  package installer used - yarn
+
+## Installing flow steps
+
+1. install babel compiler with command line interface,
+	run command <$yarn add --dev babel-cli babel-preset-flow>
+2. change the present from "react-native" to "flow"
+>	{
+>	"present": ["flow"]
+>	}
+3. Install flow version 0.65 for zulip using <$yarn add --dev flow-bin@0.65>
+4. type <$flow init> 
+5. Type flow server by typing <$yarn run flow> to check if flow is correctly installed
+
+## Uninstalling flow
```

Comment:
I don't think we need to give information how to uninstall.

Reply:
in case the developer isn't running the correct dev environment then flow doesn't work as expected. In that case a complete uninstall and reinstall of flow is required, that's why it is there.